.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         deno: ["v1.x", "nightly"]
-        os: [macOS-latest, windows-latest, ubuntu-latest]
+        os: [macOS-latest, ubuntu-latest]
```

Comment:
@lucacasonato I hope this is fine with you. Strangely the format step is failing on Windows (See https://github.com/lucacasonato/deno_local_file_fetch/runs/2226230871#step:4:283). Let's fix it now if you know the cause or let's re-introduce Windows later.